src/v/cloud_topics/CMakeLists.txt

```diff
@@ -4,9 +4,11 @@ v_cc_library(
   SRCS
     logger.cc
     types.cc
+    app.cc
```

Comment:
don't we need a corresponding change in `v/redpanda/CMakeLists.txt`?

Reply:
oh yeh didn't try the cmake build. yep, fixed.

Reply:
what cmake build? 😎